### PR TITLE
SI-4014 Scaladoc omits @author

### PR DESCRIPTION
**Pull request by Feodorov**
review by @VladUreche

### CODE REVIEW

**Comment by adriaanm** [comment]
Welcome, and thank you, @Feodorov! Please [sign the Scala CLA](http://typesafe.com/contribute/cla/scala)

**Comment by Feodorov** [comment]
I signed it a few minutes ago.

**Comment by adriaanm** [comment]
Thanks.

File: src/scaladoc/scala/tools/nsc/doc/html/HtmlPage.scala [hunk 1]
```diff
@@ -67,14 +67,20 @@ abstract class HtmlPage extends Page { thisPage =>
...
-  /** Transforms an optional comment into an styled HTML tree representing its body if it is defined, or into an empty
+  /** Transforms an optional comment into an styled HTML tree representing its body and authors (optional), or into an empty
     * node sequence if it is not. */
-  def commentToHtml(comment: Option[Comment]): NodeSeq =
-    (comment map (commentToHtml(_))) getOrElse NodeSeq.Empty
-
-  /** Transforms a comment into an styled HTML tree representing its body. */
-  def commentToHtml(comment: Comment): NodeSeq =
-    bodyToHtml(comment.body)
+  def commentToHtml(comment: Option[Comment], printAuthor: Boolean): NodeSeq =
```
Comment (VladUreche) [inline src/scaladoc/scala/tools/nsc/doc/html/HtmlPage.scala]: Wouldn't it be easier to make these changes in `Template.scala`, in `def memberToCommentBodyHtml`? There you have all the additional annotations, such as "see also", "implicit information", "migration" and you could actually just copy-paste the code for one of these annotations. As a bonus, you can access `docAuthors` as `universe.settings.docAuthors.value`, without having to carry it around. WDYT?

**Comment by VladUreche** [comment]
@Feodorov, thank you for your patch! Looking at the code, I feel it would be better to place the author generation code in `Template.scala` instead of `HtmlPage.scala`, along with all other additional annotations such as "See also", "Deprecation" and so on.

Aside from this one comment, the patch looks great!

**Comment by Feodorov** [comment]
@VladUreche Thank you for your comments, I will move changes to Template.scala, as it seems to be a much better place.

**Comment by Feodorov** [comment]
@VladUreche Here is an update for the patch.

**Comment by VladUreche** [comment]
LGTM, thank you!

**Comment by VladUreche** [comment]
@adriaanm, please merge, as I don't have commit rights :|

**Comment by adriaanm** [comment]
@VladUreche, fixed -- thanks for letting me know.

**Comment by VladUreche** [comment]
Thank you @adriaanm! ;)

**Pull request merged**